SeafShare/Info.plist

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the difference between this extension and action extension? they share a lot if code?